CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog for osjs-client
```

Comment:
This file should not have changes.

Reply:
I didn't change it, you changed it to create a search test, you told me to copy it, it was added, I didn't change it

Reply:
But this PR should **not contain this**. It's already in master.